app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt

```diff
@@ -44,19 +45,20 @@ class LoginActivity : BaseActivity() {
         })
 
         btnLogin.setOnClickListener {
-            username = tiUsername.editText?.text.toString()
-            password = tiPassword.editText?.text.toString()
-            if (validateCredentials()) {
-                loginViewModel.login(Login(username, password))
-                showProgressDialog(getString(R.string.logging_in))
-            }
+           login()
         }
 
         btnSignUp.setOnClickListener {
             intent = Intent(this, SignUpActivity::class.java)
             startActivity(intent)
             finish()
         }
+
+        tiPassword.editText?.setOnEditorActionListener { _, actionId, _ ->
+            if (actionId == EditorInfo.IME_ACTION_DONE)
```

Comment:
```suggestion
            if (actionId == EditorInfo.IME_ACTION_DONE) {
```
Wrap that `login()` instruction on brackets { }